.travis.yml

```diff
@@ -1,12 +1,13 @@
 language: python
 python:
   - "2.7"
-  - "3.4"
```

Comment:
https://github.com/googledatalab/pydatalab/pull/128 fixes the tests for Python 3, please take a look at that first.